aioshelly/rpc_device/object_mapping.py

```diff
@@ -0,0 +1,101 @@
+"""Powered by Shelly object structures."""
+
+from typing import Any
+
+OBJECT_STRUCTURES: dict[str, dict[str, Any]] = {
+    "FK-06X": {
+        "zone0": {
+            "name": "Zone 1",
+            "index": 200,
+            "type": "number",
+            "key": "duration",
+            "unit": "min",
+        },
+        "zone1": {
+            "name": "Zone 2",
+            "index": 201,
+            "type": "number",
+            "key": "duration",
+            "unit": "min",
+        },
```

Comment:
The names are set from the device and not fixed, here is an example from my device:
```json
      {
        "name": "Grass Sprinklers",
        "duration": 10,
        "water_amount": 10
      },
      {
        "name": null,
        "duration": 10,
        "water_amount": 10
      },
```

I don't understand why we hardcode the type and unit since we don't do this for any other sensor